BlogTemplate/Pages/Index.cshtml

```diff
@@ -1,15 +1,22 @@
 @page
 @model IndexModel
-@{
-    ViewData["Title"] = "Home Page";
-
-    foreach (var post in Model.PostSummaries)
-    {
+@{
+    ViewData["Title"] = "Home Page";
+
+    @if (!Model.PostSummaries.Any())
+    {
+        <h3>Welcome!</h3>
+        <p>Get started and write your first blog post <a href="/New">here.</a></p>
+        <p>Happy blogging!</p>
+    }
+
+    foreach (var post in Model.PostSummaries)

```

Comment:
this could go into an `else` block since it would only happen if the if condition was false.